autogpt_platform/backend/migrations/20250105254106_migrate_brace_to_double_brace_string_format/migration.sql

```diff
@@ -0,0 +1,78 @@
+/*
+  Warnings:
+  - You are about replace a single brace string input format for the following blocks:
+        - AgentOutputBlock
+        - FillTextTemplateBlock
+        - AITextGeneratorBlock
+        - AIStructuredResponseGeneratorBlock
+    with a double brace format.
+*/
+WITH to_update AS (
+    SELECT
+        "agentBlockId",
+        "constantInput"::jsonb AS j
+    FROM platform."AgentNode"
```

Comment:
FYI @Swiftyos 
For the marketplace, is there another table that stores an agent input that also requires a migration?